code/processes/wdb.q

```diff
@@ -43,8 +43,8 @@ hdbtypes:@[value;`hdbtypes;`hdb];                                          /-lis
 rdbtypes:@[value;`rdbtypes;`rdb];                                          /-list of rdb types to look for and call in rdb reload
 gatewaytypes:@[value;`gatewaytypes;`gateway];                              /-list of gateway types to inform at reload
 tickerplanttypes:@[value;`tickerplanttypes;`tickerplant];                  /-list of tickerplant types to try and make a connection to
-tpconnsleepintv:@[value;`tpconnsleepintv;10];                              /-number of seconds between attempts to connect to the tp
-tpcheckcycles:@[value;`tpcheckcycles;0W];                                  /-number of attempts to connect to tp before process is killed 
+/-tpconnsleepintv:@[value;`tpconnsleepintv;10];                              /-number of seconds between attempts to connect to the tp
```

Comment:
is this no longer used/needed? if so, remove it completely rather than commenting it out

why is it not used/needed anymore? does this have a negative impact on non-finspace installations?